addons/editor-extra-blocks/addon.json

```
@@ -0,0 +1,34 @@
+{
+  "name": "Reveal hidden editor blocks",
```

```suggestion
  "name": "Show hidden editor blocks",
```
Maybe even just Hidden editor blocks? The name is also fine the way it is.